src/main/bugfree/Bugfree.php

```diff
@@ -45,7 +45,7 @@ public function __construct(Resolver $resolver, Config $config)
     public function parse($name, $source)
     {
         $result = new Result($name, $this->config);
-        $parser = new Parser(new Lexer());
+        $parser = (new ParserFactory())->create(ParserFactory::ONLY_PHP7, new Lexer());
```

Comment:
Although they wouldn't be able to do that anyway because PHP would error when bugfree tried to load the file.